packages/schema/bind/src/bindings/utils.ts

```diff
@@ -25,6 +26,7 @@ export function renderTemplates(
 
           // If the file isn't empty, add it to the output
           if (data) {
+            // fs.writeFileSync(path.join("/Users/niraj/Documents/projects/polywrap/monorepo/packages/schema/bind/src/bindings/assemblyscript/wasm/templates/amodule-type", name.replace("-", ".")), data);
```

Comment:
remove?

Reply:
Yeah, that's just for debugging, I will remove it once it's ready to merge. This is helping me debug it easily.